### PR TITLE
Introduction | asherw

### PR DESCRIPTION
**Pull request by whiskerwind**
Created my intro page

### CODE REVIEW

File: intro/asherw.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<p align="center">
+<img src="img/" height="400">
+
+<h1 align="center">Asher W</h1>
+
+<h2 align="center">Student and new to Github!</h2>
+
+<p align='center'>
+  <a href="asherw88"><img height="50" src="img/gmail.png?raw=true"></a>&nbsp;&nbsp;
```
Comment (NishkarshRaj) [inline intro/asherw.md:9]: Your gmail link must be like `mailto:[id]@[domain]`

File: intro/asherw.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<p align="center">
+<img src="img/" height="400">
```
Comment (NishkarshRaj) [inline intro/asherw.md]: No Image attached!

File: intro/asherw.md (hunk 1 above)
Comment (NishkarshRaj) [inline intro/asherw.md]: Your Email format mentioned in `<a>` must be of the format `mailto:[your id]@[domain]`

**Review by NishkarshRaj — CHANGES_REQUESTED** [review]
I have added few comments. Make the mentioned changes 👍

File: intro/asherw.md (hunk 2 above)
Comment (whiskerwind) [reply intro/asherw.md]: thanks for the feedback, I figured out how to attached the image

File: intro/asherw.md (hunk 1 above)
Comment (whiskerwind) [reply intro/asherw.md]: I have fixed this link

**Review by whiskerwind — COMMENTED** [review]
I have made the changes in the review to my PR

**Review by NishkarshRaj — APPROVED** [review]
Thanks @whiskerwind for your contributions 👍

**Pull request merged**